tests/unit/test_network_connections.py

```diff
@@ -2792,13 +2792,171 @@ def test_route_v6(self):
                         "DEVICE": "e556",
                     },
                     "keys": None,
-                    "route": "192.168.40.0/24 metric 545\n192.168.46.0/30\n"
+                    "route": "192.168.40.0/24 dev e556 metric 545\n"
+                    "192.168.46.0/30\n"
+                    "192.168.45.0/24 dev e556 metric 545\n"
+                    "192.168.46.0/30 dev e556\n",
+                    "route6": "a:b:c:f::/64\na:b:c:d::/64 dev e556\n",
+                    "rule": None,
+                    "rule6": None,
+                }
+            ],
+        )
+
+    def test_route_v6_without_interface_name(self):
```

Comment:
The test name mention v6 but AFAICS, this is checking both IPv4 and IPv6.. isn't it?

Reply:
Yes, I changed the test name into `test_route_without_interface_name()`.